src/main/java/org/jitsi/jigasi/xmpp/CallControlMucActivator.java

```diff
@@ -400,6 +401,101 @@ private void updatePresenceStatusForXmppProviders()
         Statistics.updatePresenceStatusForXmppProviders();
     }
 
+    /**
+     * Adds new call control MUC xmpp account using the provided props.
+     *
+     * @param id the id to use for this account(should not contain .).
+     * @param propertiesSet the properties.
+     * @throws OperationFailedException if failed loading account.
+     */
+    public static void addCallControlMucAccount(
+            String id,
+            Set<Map.Entry<String, String>> propertiesSet)
+        throws OperationFailedException
+    {
+        ConfigurationService config
+            = JigasiBundleActivator.getConfigurationService();
+        ProtocolProviderFactory xmppProviderFactory
+            = ProtocolProviderFactory.getProtocolProviderFactory(
+                JigasiBundleActivator.osgiContext,
+                ProtocolNames.JABBER);
+        AccountManager accountManager
+            = ProtocolProviderActivator.getAccountManager();
+
+        // extract the properties, make sure we decode the password
+        HashMap<String, String> properties = new HashMap<>();
+        propertiesSet.stream().forEach(e ->
+        {
+            String property = e.getKey();
+            String value = e.getValue();
+            if (ProtocolProviderFactory.PASSWORD.equals(property))
+            {
+                value = new String(Base64.decode(value));
+            }
+
+            properties.put(property, value);
+        });
+
+        AccountID xmppAccount
+            = xmppProviderFactory.createAccount(properties);
+
+        // A small workaround to make sure we use the id for the
+        // account that is provided, this is in case we want to
+        // delete the account by the same id.
+        // We use the fact that account manager will reuse the id if found
+        // to store the properties (like edit of an account).
+        {
+            String accountConfigPrefix = accountManager
+                .getFactoryImplPackageName(xmppProviderFactory) + "." + id;
```

Comment:
If all of the other properties will use the full jitsi format, why do we treat "id" in a special way? Can't we just add the prefix in the HTTP request?